Model/Sync/Category/Processor/ProcessByCategory.php

```diff
@@ -165,6 +166,17 @@ public function processEntity($retryCategoryIds = [])
         $collection->getSelect()->limit($batchSize);
         $magentoCategories = [];
         foreach ($collection->getItems() as $category) {
+            if ($this->config->syncResetInProgress($storeId, 'catalog')) {
```

Comment:
1. The same as with products:
```
If we're skipping all of the orders of a certain store, this store shouldn't be taken into consideration in the first place.
```

2. Catalog sync should also take into consideration clearing the new `yotpo_collections_products_sync` table.

Reply:
Comment here - https://github.com/YotpoLtd/magento2-module-core/pull/136#discussion_r851813758

Reply:
We will make these changes as required